client/services/expenseTransformator.ts

```diff
@@ -0,0 +1,80 @@
+import { DebtorModel, ExpenseModel } from "./mocks/expenses";
+
+export function transformDebtorData(raw: any[]): DebtorModel {
```

Comment:
Here you can find functions that are used to convert the strange data we get from the contracts into objects.